pkgs/development/tools/ocaml/ocaml-top/default.nix

```diff
@@ -1,6 +1,11 @@
 { lib, fetchFromGitHub, ocamlPackages }:
 
-with ocamlPackages; buildDunePackage rec {
+let
+  inherit (lib) licenses maintainers;
```

Comment:
This change is not motivated: there is no “top-level `with`” that is removed there.

Reply:
I've reverted any changes having to do with `lib`.